Topics/Software_Tools/fish.md

```diff
@@ -0,0 +1,80 @@
+# A fin-troduction to `fish`
+
+## Introduction
+
+### enter `fish`!
+
+...the **f**riendly **i**nteractive **sh**ell.
+
+fish offers a command-line interface that is focused on usability and interactive use[^1].
+
+This introduction will be less about the syntax differences between the fish language
+and bash or zsh and more about the out-of-the-box features that are offered from this shell.
+
+### A dish served best straight out of the oven
+
+Some of you might be familiar with zsh plugin managers such as
+[oh-my-zsh](https://ohmyz.sh/) that can configure zsh to your liking, but fish
+just works. No more messing with chunky zsh configs and slow shells that make
+you wait several hundred milliseconds to type another command.
+
+### Installation
+
+It's recommended to check the [fish homepage](https://fishshell.com/) for up-to-date
+instructions for installing the latest version of fish on your platform.
+
```

Comment:
I installed fish using brew but noticed that your prompt looks different from the default fish prompt I have. How did you get your prompt to look like that?

Reply:
Yes! I’m using [Tide](https://github.com/IlanCosman/tide). I can add a section about plugins. That’s the only plugin I use with fish.